### PR TITLE
Fix unapplied border-radius property

### PR DESCRIPTION
**Pull request by kulikala**
The `border-radius` property of the CSS class `button.-rounded-lg` is not applied when `.ns-ios`

### CODE REVIEW

**Comment by kulikala** [comment]
@cla-bot check

File: src/scss/mixins/components/_button.scss [hunk 1]
```diff
@@ -38,15 +38,18 @@
         #{if($compat, '&-rounded-sm', '&.-rounded-sm')},
         #{if($compat, '&-rounded-lg', '&.-rounded-lg')} {
             height: const(btn-height) - 12;
+
+            .ns-android & {
+                margin: const(btn-margin-y) const(btn-margin-x);
+            }
+        }
+
+        #{if($compat, '&-rounded-sm', '&.-rounded-sm')} {
```
Comment (bundyo) [inline src/scss/mixins/components/_button.scss]: On close inspection, the small border-radius is the same as the Android default border radius of 2pt. I would rather make it the same across both OSes and remove the * 2 override completely. However, since the default small border-radius doesn't do anything on Android (compared with the default) - we might be best to up the small radius to 4 in variables/_index. Consulted with our designer and she agreed to that.

So to summarize - up $border-radius-sm to 4, remove the * 2 override completely to fix this issue. :)

**Comment by bundyo** [comment]
Thank you for the PR, check my comment.

**Comment by kulikala** [comment]
Hi, I updated `$border-radius-sm` to 4 as your comment.

File: src/scss/mixins/components/_button.scss [hunk 2]
```diff
@@ -38,17 +38,16 @@
...
-                border-radius: const(border-radius-sm) * 2;
-            }
 
             .ns-android & {
                 margin: const(btn-margin-y) const(btn-margin-x);
             }
         }
 
+        #{if($compat, '&-rounded-sm', '&.-rounded-sm')} {
+            border-radius: const(border-radius-sm);
```
Comment (bundyo) [inline src/scss/mixins/components/_button.scss]: I also meant to move back this border-radius in the upper rule, since it gets overriden in -lg anyway.

**Comment by bundyo** [comment]
Okay, I'll merge it and update it myself.

**Comment by kulikala** [comment]
@bundyo Sorry for late reply. Thank you for accepting this PR, I'm happy with it.
